nitrite.go

```diff
@@ -288,17 +312,13 @@ func Verify(data []byte, options VerifyOptions) (*Result, error) {
 		}
 	}
 
-	// Size of these fields comes from AWS Nitro documentation at
-	// https://docs.aws.amazon.com/enclaves/latest/user/enclaves-user.pdf
-	// from May 4, 2022. Experimentally verified values August 8, 2022, allow
-	// UserData limit of 3866B with a Nonce of 42B.
-	if nil != doc.PublicKey && (len(doc.PublicKey) < 1 || len(doc.PublicKey) > 1024) {
+	if nil != doc.PublicKey && len(doc.PublicKey) > maxPublicKeyLen {
 		return nil, ErrBadPublicKey
 	}
-	if nil != doc.UserData && (len(doc.UserData) < 1 || len(doc.UserData) > 1024) {
+	if nil != doc.UserData && len(doc.UserData) > maxUserDataLen {
```

Comment:
Don't enforce minimum UserData and Nonce, since they could be empty as produced by nitriding or the sequencer